homeassistant/components/media_player/__init__.py

```diff
@@ -579,6 +582,11 @@ def shuffle(self):
         """Boolean if shuffle is enabled."""
         return None
 
+    @property
+    def available_playlists(self):
```

Comment:
This is not something that we should store in the state machine. Playlists can get huge.

If Spotify knows the name, it should be enough to just send the name into playmedia

Reply:
@balloob Agreed, but my problem is, that I want to display the playlists in the media_player state card, so I need a list of the playlists somehwere. Do we have other possibilities to display dynamic data in the frontend if they are not stored in the statemachine? What should be right way?

Reply:
The current frontend does not support this, nor is it the right approach. If we want to show a list of available playlists, let's create an API call that the frontend can call to populate the list.

Reply:
@balloob I completely agree with that approach, but where I should store my data? From where the API should collect the data and deliver to the frontend?

Reply:
Just to be sure we are talking about the same - I'm storing only name and uri not the content of the playlist

Reply:
The DevChat gave me all information I need - I will implement an API call

Reply:
I will close that PR and send a new one, because I need to revert 80% of my commits I made in this PR.